src/gui_common/TweakedColourPicker.cs

```diff
@@ -194,7 +194,6 @@ public new Color Color
                 return;
 
             base.Color = value;
-            EmitSignal("color_changed", value);
```

Comment:
This had to be removed as this causes repeated signal emissions, causing problems. Also redundant because it should already be emitted under the hood.

Reply:
@84634E1A607A I think you made the tweaked colour picker, so do you have anything to say about this change?

Reply:
I recall I added this because sometimes the native color_changed signal is not emitted. (Perhaps when you change the Color property directly)

Reply:
Hm yeah, just isolated and tested this and it indeed doesn't automatically actually get emitted....

Reply:
Actually maybe it's better to not override this property, instead separate it to a new method as we sometimes want Color to not emit a signal?

Reply:
Probably yeah, but then it should be called here as well... or?

Reply:
Yes, so we will use the base `Color` and a `SetColor` method that sets `Color` and emits signal, this property should always be used in TweakedColorPicker, `Color` can be for other places.

@84634E1A607A What was actually the reason for emitting the signal by the way?

Reply:
> I think it should since that also calls this overriden property.

Yes. (I said once, when this property is not overriden.) And this is why I chose to emit a signal here.

Reply:
Yes, but I didn't playtest so I'm not sure all functions are still correct.

Reply:
I'd prefer a `if (Color == colour) return;` added